recipes/sdl_ttf/all/conanfile.py

```diff
@@ -82,22 +98,28 @@ def build(self):
 
     def package(self):
         copy(self, "COPYING.txt", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        copy(self, "LICENSE.txt", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
         cmake = CMake(self)
         cmake.install()
         rmdir(self, os.path.join(self.package_folder, "cmake"))
         rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
         rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
         rmdir(self, os.path.join(self.package_folder, "SDL2_ttf.framework"))
+        rmdir(self, os.path.join(self.package_folder, "share"))
 
     def package_info(self):
+        suffix = "-static" if Version(self.version) >= "2.20.0" and not self.options.shared else ""
+
         self.cpp_info.set_property("cmake_file_name", "SDL2_ttf")
-        self.cpp_info.set_property("cmake_target_name", "SDL2_ttf::SDL2_ttf")
+        self.cpp_info.set_property("cmake_target_name",f"SDL2_ttf::SDL2_ttf{suffix}")
         self.cpp_info.set_property("pkg_config_name", "SDL2_ttf")
 
         self.cpp_info.includedirs.append(os.path.join("include", "SDL2"))
         self.cpp_info.libs = ["SDL2_ttf"]
```

Comment:
- `-static` suffix for MSVC
- There is also a `d` suffix for debug libraries

A heads-up, the `d` suffix will be removed from SDL3.
OTOH, by passing `-DSDL2TTF_DEBUG_POSTFIX=""` to cmake configure, you can disable the debug suffix.

```suggestion
        self.cpp_info.libs = [f"SDL2_ttf{'-static' if is_msvc(self) and not self.options.shared}{'d' is self.settings.build_type == "Debug' else ''}"]
```